src/services_models/services_constructor.jl

```diff
@@ -72,3 +72,32 @@ function construct_service!(
     end
     return
 end
+
+function construct_service!(
+    psi_container::PSIContainer,
+    services::IS.FlattenIteratorWrapper{PSY.AGC},
+    services_mapping::PSY.ServiceContributingDevicesMapping,
+    ::ServiceModel{PSY.AGC, T},
+    devices_template::Dict{Symbol, DeviceModel},
+) where {T <: AbstractAGCFormulation}
+    #Note: Frequency is a global variable
```

Comment:
I don't follow the comment.

Reply:
The Frequency of the system is the same all over not just for an area.